app/Http/Controllers/DirectCostEstimateController.php

```
@@ -0,0 +1,36 @@
+namespace App\Http\Controllers;
+
+use App\Http\Requests\FilterDirectCostEstimateRequest;
+use App\Http\Resources\DirectCostEstimateResource;
+use App\Models\ResourceItem;
+
+class DirectCostEstimateController extends Controller
+{
+    public function index(FilterDirectCostEstimateRequest $request)
+    {
+        $validated = $request->validated();
+        $task_id = $validated['task_id'] ?? null;
```

PHPMD (2.15.0)

The variable $task_id is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)